duckduckgo/duckduckgo-privacy-extension
# Clean up old grade details files and references

**Reviewer:** @andrey-p 



## Description:
Removing unused legacy code (view and template) for the grade details subview.


## Steps to test this PR:
1. Build and reload extension
2. Make sure the grade details work as expected and nothing is missing


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
